src/network_scripts.rs

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) 2023, MaidSafe.
```

Comment:
Hey Roland, why do we call these "scripts" if they are written in Rust? What distinguishes them from other Rust code?